cmd/app/main.go

```diff
@@ -0,0 +1,16 @@
+package main
```

Comment:
I suggest you choosing a different name for this package, so it is more descriptive when you `go build` it